mParticle-iterable/MPKitIterable.m

```diff
@@ -55,7 +57,7 @@ - (BOOL)handleIterableURL:(NSURL *)url context:(IterableActionContext *)context
     attributionResult.linkInfo = getAndTrackParams;
 
     [self->_kitApi onAttributionCompleteWithResult:attributionResult error:nil];
-    return result;
+    return true;
```

Comment:
Why ignore `result` and always return `true` (which should be `YES` since this is a `BOOL` type)?